sdk/keyvault/keyvault-keys/test/public/crypto.spec.ts

```diff
@@ -107,6 +107,7 @@ describe("CryptographyClient (all decrypts happen remotely)", () => {
     const hash = createHash("sha256");
     hash.update(signatureValue);
     const digest = hash.digest();
+    console.log({ digest });
```

Comment:
this seems like something not meant to be left in?

Reply:
Oh my bad, thank you!